.github/workflows/checks.yml

```diff
@@ -55,7 +55,7 @@ jobs:
 
       - name: Run changelog update check
         if: ${{ github.ref != 'refs/heads/main' }}
-        run: poetry run nox -s changelog:updated
+        run: poetry run -- nox -s changelog:updated
```

Comment:
The change in the -- usage is discussed in https://github.com/python-poetry/poetry/issues/10204.